CONTRIBUTING.md

```diff
@@ -28,11 +30,91 @@ You can then make vs code use this environment easily with `ctrl + shift + p` se
 5) Verify that your poetry is now running, using the command `poetry --version`.
 
 
+## How to contribute to the Provena Python Client
+
+### 1) Initial Setup
+
+1. **Ensure You Have an IDE Installed**: Start by setting up your preferred Integrated Development Environment (IDE) like VSCode, PyCharm, etc.
+2. **Clone the Repository**: Use the following command to clone the Provena client repository locally: `git clone https://github.com/provena/provena-python-client.git `
+3. **Install Requirements**: The Provena Python client uses Poetry for dependency management. Install Poetry if it's not already installed, following the steps above. 
+4. **Set Up Mypy for Static Type Checking**: Ensure you have Mypy configured to check for type errors. You can run Mypy manually: `poetry run mypy` or `python run mypy` if using local .venv
+
+### 2) Codebase Layout and Contributing
+
+Before making any contributions, it's important to understand the core structure of the Provena Python Client.
+
+- **Core Directory Layout**:
+
+    - **docs**: All documentation is located here and should be updated if your contribution involves documentation changes or additions.
+    - **src/provenaclient**: This directory contains the core functionality of the Provena client. Any new features, bug fixes, or changes to the core logic should be made here.
+    - **tests**: All unit and integration tests are located in this directory. If your contribution involves code changes or new features, corresponding tests should be written or updated here.
+    
+- **Where to Add New Features**:
+
+    - Any new feature should be added within the `src/provenaclient/` directory, following the existing architecture structure and file organisation.
```

Comment:
Nice, can you add a "see readme in src/ for more information to this sentance.

Reply:
Sure sounds good.

Reply:
@Rosspet added a dot point instead. Let me know what you think.